exporter/opentelemetry-exporter-jaeger-proto-grpc/src/opentelemetry/exporter/jaeger/proto/grpc/__init__.py

```diff
@@ -122,11 +123,11 @@ def __init__(
         self.collector_endpoint = collector_endpoint or environ.get(
             OTEL_EXPORTER_JAEGER_ENDPOINT, DEFAULT_GRPC_COLLECTOR_ENDPOINT
         )
+        self.insecure = insecure or environ.get(OTEL_EXPORTER_JAEGER_GRPC_INSECURE)
```

Comment:
This can be boolean, string, None. Make the result to be just boolean. 

```suggestion
        self.insecure = insecure or environ.get(OTEL_EXPORTER_JAEGER_GRPC_INSECURE, "").strip().lower() == "true"
```